### ISSUE TITLE
Why do we have 128 neurons for model in examples/addition_rnn.py

### ISSUE THREAD

**Issue by Arsey** [fchollet/keras#2829]
It would be good probably to have a comment of why 128 neurons have been chosen in the example https://github.com/fchollet/keras/blob/master/examples/addition_rnn.py.
Because the given model converges on:
_One layer LSTM (60 HN), 50k training examples = 99% train/test accuracy in 100 epochs_

**Comment by keunwoochoi** [comment]
I believe such hyperparameters in the examples are rather arbitrary, which is totally okay because it's just example. Probably the only thing we could comment would be, therefore, `# arbitrary`..
